scripts/page_browse.js

```diff
@@ -331,7 +331,8 @@ function pageBrowse(params, replaceUrl, mentorMode = false) {
                 params.set("imagefile", imageFileName);
                 document.title = proofIntData.strings.displayPageX.replace("%s", imageFileName);
                 if(displayMode !== "text") {
-                    imageElement.attr("src", page.image_url);
+                    imageElement.attr("src", page.image_url).parent()
+                        .scrollTop(0);
```

Comment:
this is only vertical position, right? Do we want to reset horizontal scroll as well?

Reply:
Oh, good point. The prior behavior is "scroll to the top left", but is that what people would want/expect? People may want to zoom + horiz scroll to remove the margins and have that persist across pages.

Although changing it so that it scrolls top and left is probably the path of least resistance at this point in time.

Reply:
Good point, I'll look at it also.

Reply:
Top left as a default makes the most sense to me. We're unlikely to please everyone, but I would think that's what most people would expect, if not for it to stay in the same position as the previous page.

Reply:
Good point, I'll look at it also.

Reply:
Implemented now in the sandbox.